db/migrate/20160304025530_add_overdue_count_to_categories_and_eq_models.rb

```diff
@@ -0,0 +1,16 @@
+class AddOverdueCountToEquipmentModels < ActiveRecord::Migration
+
+  def self.up
+    add_column :equipment_models, :overdue_count, :integer, :null => false, :default => 0
+
+    EquipmentModel.each do |e|
```

Comment:
You need to write these as raw SQL queries to avoid making your migration dependent on any current Models. See #1393 for an example of how to do that.